iron-fit-behavior.html

```diff
@@ -485,14 +485,19 @@
     },
 
     __getNormalizedRect: function(target) {
-      if (target === document.documentElement || target === window) {
+      if (target === window) {
```

Comment:
Oh, actually, I see what they're going for in that situation. We shouldn't 'correct' for that case: pinch and zoom aren't meant to change the logical viewport, they're meant change what area of the logical viewport is displayed by the physical viewport. Consider OS X's ctrl+scroll zooming; it would be really broken if, for example, all windows on screen were set up to stay within the visible zoomed area (rather than the logical bounds of the screen) and moved / resized themselves to fit within it.

I tried to repro #61 and it's almost always correct as is; I wonder why it's occasionally flaky though.